modules/core/src/system.cpp

```diff
@@ -114,10 +114,10 @@ void* allocSingletonNewBuffer(size_t size) { return malloc(size); }
 #include <cstdlib>        // std::abort
 #endif
 
-#if defined __ANDROID__ || defined __linux__ || defined __FreeBSD__ || defined __OpenBSD__ || defined __HAIKU__
+#if defined __ANDROID__ || defined __unix__ || defined __FreeBSD__ || defined __OpenBSD__ || defined __HAIKU__
 #  include <unistd.h>
 #  include <fcntl.h>
-#  include <elf.h>
+#  include <sys/elf.h>
```

Comment:
This change causes an build error on Linux x64 / Android ARM builds.

Reply:
I'll look into it.

Reply:
Fixed in cd306537f92676a741712511bfecd4336fa851cf.